service/installer/helm/helm.go

```diff
@@ -215,29 +215,18 @@ func (i *Installer) Install(project spec.Project) error {
 	return nil
 }
 
-func (i *Installer) List(projects []spec.Project) ([]spec.Project, error) {
+func (i *Installer) List() ([]spec.Project, error) {
```

Comment:
The interface is nicer now on the first glance. This makes more sense to me. The disappointing thing is the returned result. More on this below.